README.md

```diff
@@ -53,4 +53,6 @@ Options:
   -cpus                 Number of used cpu cores.
                         (default for current machine is 8 cores)
   -host                 HTTP Host header.
+  -http-trace         	Enable http trace detailed info on various events during request (Experimental)
```

Comment:
httptrace is an terminology that makes more sense in the scope of the Go standard library.

Should we just rename this flag to reflect that it is providing just more info? A flag like -more?